project/Build.scala

```diff
@@ -205,8 +238,9 @@ object Scrooge extends Build {
     name := "scrooge-generator",
     libraryDependencies ++= Seq(
       "org.apache.thrift" % "libthrift" % libthriftVersion,
-      "com.github.scopt" %% "scopt" % "3.3.0",
-      "com.github.spullara.mustache.java" % "compiler" % "0.8.18",
```

Comment:
nope, no direct reason, though it is possible if that's the path you want to take. i was just addressing your comment here: https://github.com/twitter/scrooge/issues/236#issuecomment-220889358